### DISCUSSION TITLE
Commerce SDK Migration

### DISCUSSION THREAD

**Discussion by pruthvigowda483** [microsoft/Dynamics365Commerce.Solutions#195] [Q&A] [answered]
### Summary
**Describe the issue**
A clear and concise description of the issue you're experiencing with the Commerce SDK Sample

**Version and Error Info**
Sample Version:

**Extension Requirement**
Please provide a concise description of the scenario you are trying to enable with this extension. This information is helpful for us to determine if there is an alternative implementation that avoids the issue.

### Screenshots
Please include any screenshots that would help us understand the issue.

**Comment by pruthvigowda483** [comment]
Hi Team, i am migrating the Retail SDK to Commerce SDK but i am struck in POS solutions, i have few doubts can anyone help me out here
1) How to debug the POS solutions from commerce SDK(In Retail we will run directly the MPOS solutions) but in commerce SDK we cannot do that
2) How to test POS extensions in Store commerce.
2) I can see 2 samples in github  ScaleunitSample and InStore sample, should we have to use both or can i use only one that is InStore Sample inside that can i write both CRT samples and POS extensions

Thanks in advance

**Reply by madyke** [reply, marked as answer]
Hello @pruthvigowda483, this document should help you with your first two questions, getting started using VS Code to debug Store Commerce and extensions:
[Debug Store Commerce extensions using Visual Studio Code - Commerce | Dynamics 365 | Microsoft Learn](https://learn.microsoft.com/en-us/dynamics365/commerce/dev-itpro/sc-debug)

For question 3, you can have both CRT and POS extensions in the same solution. The **PosExtensionSamples** sample should provide an example of doing this, where it has POS extensions inthe `Pos` folder, and CRT extensions (in this case just to load the POS extensions) in the `CommerceRuntime` folder.
[Dynamics365Commerce.InStore/src/StoreCommerceSamples/PosExtensionSamples at release/9.47 · microsoft/Dynamics365Commerce.InStore (github.com)](https://github.com/microsoft/Dynamics365Commerce.InStore/tree/release/9.47/src/StoreCommerceSamples/PosExtensionSamples)
